libs/ColorKit/include/ColorPosition.h

```diff
@@ -0,0 +1,18 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <algorithm>
+#include <cstdint>
+
+#include "RGB.h"
+
+namespace leka {
+
+struct ColorPosition {
```

Comment:
j'ai du mal à comprendre ce à quoi ça correspond. j'imagine que tu veux parler de la couleur correspondante à une position dans le gradient entre 0 et 255?

dans ce cas, je dirai plutôt:

```suggestion
struct GradientSpotColor {
```